spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe Users::SessionsController, type: :controller do
+  describe "POST #create" do
+    before :each do
+      @request.env["devise.mapping"] = Devise.mappings[:user]
+      @user = FactoryGirl.create :user
+    end
+
+    context "when the credentials are correct" do
+      before :each do
+        credentials = { email: @user.email, password: '123456' }
+        post :create, params: { session: credentials }
```

Comment:
@valentina91p I have tried that, but the test fails with the following message: 

`Failure/Error: params.require(:session).permit(:email, :password)
     ActionController::ParameterMissing:
       param is missing or the value is empty: session`

Using params was the only way I found to make the tests  work. The endpoint works fine using just the session when I test it with postman, but in RSpec I had to use params.